cli/command/container/opts.go

```diff
@@ -825,6 +832,23 @@ func parseSecurityOpts(securityOpts []string) ([]string, error) {
 	return securityOpts, nil
 }
 
+func parseSystemPaths(securityOpts []string) ([]string, []string, []string, error) {
```

Comment:
I realise this function is not exported, but it might be important to document what it does (especially because it filters the `securityOpts`; perhaps add a short GoDoc; something along the line of:

```suggestion
// parseSystemPaths checks if `systempaths=unconfined` security option is set,
// and returns the `MaskedPaths` and `ReadonlyPaths` accordingly. An updated
// list of security options is returned with this option removed, because the
// `unconfined` option is handled client-side, and should not be sent to the
// daemon.
func parseSystemPaths(securityOpts []string) ([]string, []string, []string, error) {
```

(suggestions for better wording welcome! 😂)


Although I'm generally not a fan of named return variables, I'm a bit on the fence if we should use them here (for readability), as three `[]string` returns might be difficult to read/grasp;

```suggestion
func parseSystemPaths(securityOpts []string) (filtered, maskedPaths, readonlyPaths []string) {
```

I'd love to get feedback from others in that as well though 😅